nativelib/src/main/resources/scala-native/gc/commix/Synchronizer.c

```diff
@@ -1,6 +1,7 @@
 #if defined(SCALANATIVE_MULTITHREADING_ENABLED) &&                             \
     defined(SCALANATIVE_GC_COMMIX)
 
+#include "../constants.h"
```

Comment:
This can be just `"constants.h"`. If you look at `compile_flags.txt` you can that both `.` and `gc` are added to the include path. This of course is for using `clangd` (LSP) - See https://scala-native.org/en/stable/contrib/ides.html#setup-for-clangd

The real include path sent to the Scala Native compiler (tools) is in the `scala_native.properties` in each project with native code - `resources/scala_native` dir. The default is just to have `.` in the include path but others can be added via that special properties file. This was added because `nativelib` had some special rules and we had a lot of confusing relative paths in the includes. Hope this helps.

Reply:
Oh, indeed  I had the impression that I got that wrong (after all, shouldn0t it be `../../`?), but it worked, so I ignored it.
Will change it.

Reply:
Changed in [c191bc0](https://github.com/scala-native/scala-native/pull/4310/commits/c191bc0c1cfa1ce9a5ef0586d0421f20267185f0)